tests/e2e/support/objects/app-files/page/shares/withMe.ts

```diff
@@ -1,5 +1,7 @@
 import { Page } from 'playwright'
 
+const sharesOptionSelector = '//a[@data-nav-name="files-shares-with-me"]'
```

Comment:
looks like the selector is not options but  a nav link
```suggestion
const shareWithMeNavSelector = '//a[@data-nav-name="files-shares-with-me"]'
```